src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -109,34 +124,41 @@ public static TensorFlowTransform Create(IHostEnvironment env, ModelLoadContext
             Contracts.CheckValue(env, nameof(env));
             env.CheckValue(ctx, nameof(ctx));
             ctx.CheckAtModel(GetVersionInfo());
+
             // *** Binary format ***
+            // int: indicator for frozen models
             // stream: tensorFlow model.
             // int: number of input columns
             // for each input column
             //   int: id of int column name
             // int: number of output columns
             // for each output column
             //   int: id of output column name
-            byte[] modelBytes = null;
-            if (!ctx.TryLoadBinaryStream("TFModel", r => modelBytes = r.ReadByteArray()))
-                throw env.ExceptDecode();
-            var numInputs = ctx.Reader.ReadInt32();
-            env.CheckDecode(numInputs > 0);
-            string[] inputs = new string[numInputs];
-            for (int j = 0; j < inputs.Length; j++)
-                inputs[j] = ctx.LoadNonEmptyString();
+            var isFrozen = ctx.Reader.ReadInt32();
+            if (isFrozen == 1)
+            {
+                byte[] modelBytes = null;
+                if (!ctx.TryLoadBinaryStream("TFModel", r => modelBytes = r.ReadByteArray()))
+                    throw env.ExceptDecode();
 
-            bool isMultiOutput = ctx.Header.ModelVerReadable >= 0x00010002;
-            var numOutputs = 1;
-            if (isMultiOutput)
-                numOutputs = ctx.Reader.ReadInt32();
+                var io = ModelInputsOutputs(env, ctx);
+                return new TensorFlowTransform(env, modelBytes, (isFrozen == 1), io.Item1, io.Item2);
+            }
+            else
+            {
+                // Load model binary
```

Comment:
>/ Load model binary [](start = 17, length = 19)

for un-frozen models we need to extract the folder structure that was retained during Save(). Here I am :

(a) Extracting the folder structure from the byte[] into a temporary folder
(b) Creating a TFSession using the extracted folder structure
(3) Deleting the temporary folder 
(4) Invoking the constructor to create an instance of the TFTransform